README.md

```diff
@@ -123,6 +123,12 @@ You can include the library as a regular script tag on your page:
 <script src="dist/onfido.min.js"></script>
 ```
 
+⚠️ Note: The above import does **not** include the Auth module. To include it, use:
+
+```html
+<script src="dist/onfidoAuth.min.js"></script>
```

Comment:
Would it be worth moving this further up and just having it as part of the same code snippet as `<script src="dist/onfido.min.js"></script>`?